DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Delegates/DelegateGroups/GroupCourseExpandableViewModel.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.DelegateGroups
+{
+    using DigitalLearningSolutions.Data.Models.DelegateGroups;
+
+    public class GroupCourseExpandableViewModel

```

Comment:
I think we can drop the "Expandable" from the name here + for delegate VC as it doesn't add much, but makes it more of a mouthful to read

Reply:
Happy with revereting to GroupCourseViewModel (for cards) and GroupCoursesViewModel (for the page) then?